### PR TITLE
Practice3

### PR DESCRIPTION
**Pull request by stratopedarx**
Release all mandatory requirements

### CODE REVIEW

File: practice_2/StarWarsApp/StarWarsVC.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return heroModels.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        // swiftlint:disable force_cast
+        let cell = starWarsTableView.dequeueReusableCell(withIdentifier: HeroCell.identifier,
+                                                         for: indexPath) as! HeroCell
```
Comment (alekseykorol) [inline practice_2/StarWarsApp/StarWarsVC.swift]: Force casts should be avoided ([Force Cast](https://realm.github.io/SwiftLint/force_cast.html)).

File: practice_2/StarWarsApp/cells/HeroCell.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import UIKit
+
+class HeroCell: UITableViewCell {
+    static let identifier = "heroCell"
+    @IBOutlet weak var heroNameLabel: UILabel!
```
Comment (alekseykorol) [inline practice_2/StarWarsApp/cells/HeroCell.swift]: IBOutlets should be private to avoid leaking UIKit to higher layers

Comment (stratopedarx) [reply practice_2/StarWarsApp/cells/HeroCell.swift]: Done. Thanks.

File: practice_2/StarWarsApp/StarWarsVC.swift (hunk 1 above)
Comment (stratopedarx) [reply practice_2/StarWarsApp/StarWarsVC.swift]: Thanks Aleksey.
I will take into account all your comments in future projects

**Review by vasilevkin — APPROVED** [review]

**Pull request merged**